packages/dendron-cli/src/commands/devCLICommand.ts

```diff
@@ -225,6 +234,17 @@ export class DevCLICommand extends CLICommand<CommandOpts, CommandOutput> {
           await this.syncAssets(opts);
           return { error: null };
         }
+        case DevCommands.SYNC_TUTORIAL: {
+          if (!opts.treatmentName) {
```

Comment:
is there every a case where we don't just want to sync all the tutorials? anything under `tutorial*` should be valid